tools/icu/icu-generic.gyp

```diff
@@ -401,6 +441,30 @@
           '../../deps/icu/source/common/uts46.cpp',
           '../../deps/icu/source/common/uidna.cpp',
         ]}],
+        [ 'icu_ver_major == 57', { 'sources!': [
+          ## Strip out the following for ICU 55 only.
+          ## add more conditions in the future?
```

Comment:
thanks, refactor and removed the todo. (it's done now that there are 57 AND 55 clauses)